.circleci/config.yml

```diff
@@ -91,6 +91,12 @@ jobs:
           command: |
             make vendor
             hack/tree_status.sh
+      - run:
+          name: check profile consistency
+          command: |
+            make default.json
```

Comment:
Do we want to adapt the make target to directly overwrite `seccomp.json`. To me it was a bit confusing having both laying (locally) around within the repo.

Reply:
Nice idea, I'll add that to this PR.